### PR TITLE
about us page created and route created on the routes table

### PR DESCRIPTION
**Pull request by oluolofinyo**
Task Screenshot:

about-mobile
about-desktop

### CODE REVIEW

**Review by shonubijerry — CHANGES_REQUESTED** [review]
You did not push the `about.ejs` file that you created

**Comment by oluolofinyo** [comment]
file added...

File: src/routes/routes.js [hunk 1]
```diff
@@ -41,7 +41,7 @@ export const initRoutes = app => {
   app.get('/post', (req, res) => res.render('userPost', { isAuth: true }));
   app.get('/jobInvites', renderJobInvitesPage);
   app.get('/post/:inviteId', renderSinglePostPage);
-  app.get('/admin/reported', (req, res) => res.render('admin/reportedUsers', { isAuth: true, }));
```
Comment (shonubijerry) [inline src/routes/routes.js:54]: Why did you remove this line and replace with about?

**Review by shonubijerry — CHANGES_REQUESTED** [review]
Avoid changing codes form develop branch

File: src/routes/routes.js (hunk 1 above)
Comment (oluolofinyo) [reply src/routes/routes.js:54]: Oh, GitBash auto-merged that while I did an upstream pull I believe.
Route was already defined on that line. If you check my initial PR commit 
[a294cea](https://github.com/hngi/is-this-a-real-job/pull/79/commits/a294cea05e9ee967675e5f8d48bf2a6bf7d37462)

'/admin/reported', Route will be redefined on the line right after it in my next PR.

**Review by shonubijerry — APPROVED** [review]
